hera_mc/cm_hookup.py

```diff
@@ -57,11 +57,14 @@ def get_hookup(self, hpn, rev, port, at_date, state_args, exact_match=False):
         Parameters
         -----------
         hpn:  the input hera part number (whole or first part thereof)
-        rev_:  the revision number
-        port_:  a specifiable port name,  default is 'all'.  Unverified.
+        rev:  the revision number
+        port:  a specifiable port name,  default is 'all'.  Unverified.
         at_date:  date for hookup validity
-        state_args:  keyword dictionary specifying parameters for hookup
         exact_match:  boolean for either exact_match or partial
+        kwargs:  optional arguments to allow for inserting levels
+                    show_levels:  boolean to include levels
+                    levels_testing:  if present and not False will pull levels from
+                                     the file given in levels_testing
```

Comment:
Why are these options accessed through `**kwargs` and not simply two new keyword arguments defaulting to `False`? From reading the function I can't see a reason for this convoluted approach.